### ISSUE TITLE
Add the ability to create links that can launch a specific game

### ISSUE THREAD

**Issue by khrome23** [FunKey-Project/gmenu2x#5]
I was trying to add links that can launch specific games using examples I found in documentation and discussions of other ports of gmenu2x.  When I was unsuccessful, I posted a question on reddit and was told that this port doesn’t allow us to do that.  So I would like to submit is as a request.  :-). Thanks for your consideration!

**Comment by DrUm78** [comment]
This is already possible, just click on Select (Fn) and select "Add link in <SECTION_NAME>" then you can select what app/binary you want to launch with this new link.

**Comment by khrome23** [comment]
Yes, I was able to launch an bin.  But what I want to do is launch an   emu and have it open a specific rom, so I can make the link and icon be the name of the game.  Is this even possible?  Thanks for your expertise!  :-D

**Comment by DrUm78** [comment]
Indeed that's not possible and not planned on my side.
